action.yaml

```diff
@@ -354,6 +365,11 @@ runs:
         cd ${{ inputs.root-folder }}
         astro deploy ${{steps.deployment-preview.outputs.FINAL_DEPLOYMENT_ID}} ${{steps.deploy-options.outputs.OPTIONS}}
       shell: bash
+    - name: Remove override on Deployment to resume schedule
```

Comment:
You may want to move this step to happen after "Copy Airflow Connections, Variables, and Pools". That step requires the deployment to be awake too